### DISCUSSION TITLE
syntax error  when i install kali nethunter

### DISCUSSION THREAD

**Discussion by sharmasheesh** [community/community#146156] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

./install-nethunter-termux: line 107: syntax error near unexpected token `}'

**Comment by fansarq** [comment]
quick solution for the syntax error on line 107:

Redownload the script to ensure it's not corrupted:

wget -O install-nethunter-termux https://offs.ec/2MceZWr
chmod +x install-nethunter-termux
./install-nethunter-termux

Run it with Bash:
bash ./install-nethunter-termux

Check line 107: Open the script, locate line 107, and verify if there’s an unexpected }. Adjust the syntax if necessary.
